__tests__/app.test.js

```diff
@@ -37,6 +38,41 @@ describe("/api/topics", () => {
   });
 });
 
+describe('/api/articles/:article_id', () => {
+    test("GET:200 sends a single article to the client", () => {
+        return request(app)
+          .get("/api/articles/2")
+          .expect(200)
+          .then((response) => {
+            const article = response.body.article
+              expect(typeof article.author).toBe("string");
+              expect(typeof article.title).toBe("string");
+              expect(typeof article.article_id).toBe("number");
```

Comment:
As we know the specific article ID here (2), we can change this expect statement to test for it 👍